backend/src/routers/gameRouter.ts

```diff
@@ -267,16 +270,60 @@ gameRouter.delete('/:id', async (req, res) => {
  *    "message": "Game marked as played"
  *   }
  *
+ * @apiErrorExample {json} Unauthorized:
+ * HTTP/1.1 401 Unauthorized
+ *   {
+ *    "message": "Unauthorized"
+ *   }
+ *
  * @apiUse ZodError
  */
-gameRouter.post('/markPlayed', async (req, res) => {
+gameRouter.post('/markPlayed/:gameId', async (req, res) => {
 	try {
-		if (!req.user) return res.status(401).json({ message: 'Unauthorized' });
-		await markGameAsPlayed(req.body.gameId, (req.user as GammaUser).cid);
+		if (!req.isAuthenticated()) return res.status(401).json({ message: 'Unauthorized' });
```

Comment:
There's a middleware for this function as I understand it. You should probably use that instead.